pkg/registry/event/rest.go

```diff
@@ -45,7 +46,8 @@ func NewStorage(registry generic.Registry) *REST {
 func (rs *REST) Create(ctx api.Context, obj runtime.Object) (runtime.Object, error) {
 	event, ok := obj.(*api.Event)
 	if !ok {
-		return nil, fmt.Errorf("invalid object type")
+		glog.V(2).Infof("Invalid object type. Was expecting an object of type api.Event: %v", obj)
```

Comment:
Ordinarily I'd agree, but this is a programmer error and we're passing it as an "InternalError" so I think it's OK. Plus this way when people complain about it on the mailing list, we'll be able to investigate directly without asking for logs.